### ISSUE TITLE
Sweep: DoG+Global+Cross Entropy, sweeping hyperparameters excluding RF, RF ratio

### ISSUE THREAD

**Issue by RylanSchaeffer** [RylanSchaeffer/FieteLab-No-Free-Lunch#2]
Experiment: Train various models on Difference of Gaussian receptive fields with global normalization and cross entropy loss, holding the receptive field (RF) and RF ratio fixed at 0.12 and 2.0  respectively (values chosen by Aran, maybe also Ben).

W&B Page: https://wandb.ai/rylan/mec-hpc-investigations/sweeps/sp2hvkth?workspace=user-rylan

Warning: This sweep needs to be run again for much longer (probably 2-3x) because DoG converge more slowly than Gaussians.

Results:

- DoG seems slower to learn than Gaussian. 20 epochs with 500 grad steps per epoch doesn't always converge yet. Training for longer looks promising though

![image](https://user-images.githubusercontent.com/8942987/161095856-ec1c1efa-9349-4081-b0c8-69a695e5d6f0.png)

![image](https://user-images.githubusercontent.com/8942987/161095711-94153403-7b90-4806-973d-aa1c3e9a0b91.png)

- Many models learn d60 grid cells
![image](https://user-images.githubusercontent.com/8942987/161095948-075dbc3d-d9eb-46bd-817b-c98c7d503b42.png)

- Many models also learn d90 grid cells, on the same time scale
![image](https://user-images.githubusercontent.com/8942987/161096116-b0ac82eb-9a2e-496a-ace5-1410c7503371.png)

- As the models learn, the embedding dimensionality increases. My recollection is that this is also true for the Gaussian targets; however, for DoG
![image](https://user-images.githubusercontent.com/8942987/161097007-26b78c1b-dbd6-4ec9-9d2b-7aa3a5ec8373.png)
